ui/src/app/views/workspace-list/component.ts

```diff
@@ -68,15 +71,7 @@ export class WorkspaceListComponent implements OnInit, OnDestroy {
   queryBillingStatus(): void {
     // TODO (blrubenstein): When we have a home page, move this to the
     //      home page and change from tooltip to more descriptive message.
-    this.profileService.getMe().subscribe((profile: Profile) => {
-      if (profile.freeTierBillingProjectStatus === BillingProjectStatus.Ready) {
-        this.billingProjectInitialized = true;
-      } else {
-        this.billingProjectQuery = setTimeout(() => {
-          this.queryBillingStatus();
-        }, 10000);
-      }
-    });
+    this.profileStorageService.requestNewProfile();
```

Comment:
The wrapper method doesn't seem necessary any longer, would just inline this above.